Source/WebInspectorUI/UserInterface/Views/ThreadTreeElement.js

```diff
@@ -45,6 +45,8 @@ WI.ThreadTreeElement = class ThreadTreeElement extends WI.GeneralTreeElement
         this._updateStatus();
 
         let targetData = WI.debuggerManager.dataForTarget(this._target);
+        if (!targetData)
+            return;
```

Comment:
should all of these eventually be removed?  if so, please add FIXME comments just like above